src/Components/CreateImageWizard/formComponents/TargetEnvironment.js

```diff
@@ -32,129 +37,203 @@ const TargetEnvironment = ({ label, isRequired, ...props }) => {
+            title="Amazon Web Services"
+            icon={
+              <img
+                className="provider-icon"
+                src={'/apps/frontend-assets/partners-icons/aws.svg'}
+              />
+            }
+            onClick={() => handleSetEnvironment('aws', !environment.aws)}
+            onKeyDown={(e) => handleKeyDown(e, 'aws', !environment.aws)}
+            onMouseEnter={() => prefetchSources('aws')}
+            isSelected={environment.aws}
+            isStacked
+            isDisplayLarge
           />
-        </FormGroup>
-        <FormGroup
-          label={<Text component={TextVariants.small}>Other</Text>}
-          data-testid="target-other"
-        >
-          <Checkbox
-            label="Virtualization - Guest image (.qcow2)"
-            isChecked={environment['guest-image']}
-            onChange={() => handleSetEnvironment('guest-image')}
-            aria-label="Virtualization guest image checkbox"
-            id="checkbox-guest-image"
-            name="Virtualization guest image"
-            data-testid="checkbox-guest-image"
+          <Tile
+            className="tile pf-u-mr-sm"
+            data-testid="upload-google"
+            title="Google Cloud Platform"
+            icon={
+              <img
+                className="provider-icon"
+                src={
+                  '/apps/frontend-assets/partners-icons/google-cloud-short.svg'
+                }
+              />
+            }
+            onClick={() => handleSetEnvironment('gcp', !environment.gcp)}
+            isSelected={environment.gcp}
+            onKeyDown={(e) => handleKeyDown(e, 'gcp', !environment.gcp)}
+            isStacked
+            isDisplayLarge
           />
-          <Checkbox
-            label="Bare metal - Installer (.iso)"
-            isChecked={environment['image-installer']}
-            onChange={() => handleSetEnvironment('image-installer')}
-            aria-label="Bare metal installer checkbox"
-            id="checkbox-image-installer"
-            name="Bare metal installer"
-            data-testid="checkbox-image-installer"
+          <Tile
+            className="tile pf-u-mr-sm"
+            data-testid="upload-azure"
+            title="Microsoft Azure"
+            icon={
+              <img
+                className="provider-icon"
+                src={
+                  '/apps/frontend-assets/partners-icons/microsoft-azure-short.svg'
+                }
+              />
+            }
+            onClick={() => handleSetEnvironment('azure', !environment.azure)}
+            onKeyDown={(e) => handleKeyDown(e, 'azure', !environment.azure)}
+            onMouseEnter={() => prefetchSources('azure')}
+            isSelected={environment.azure}
+            isStacked
+            isDisplayLarge
           />
-        </FormGroup>
+        </div>
       </FormGroup>
-    </>
+      <FormGroup
+        label={<Text component={TextVariants.small}>Private cloud</Text>}
```

Comment:
Can we put slightly more space between the public cloud buttons and the "Private cloud" text?

Reply:
I added a `pf-u-mt-sm`, lmk if you think `pf-u-mt-md` would be better

Reply:
`pf-u-mt-sm` feels right, thanks!